samplers.qmd

```diff
@@ -0,0 +1,113 @@
+# Samplers and inference {#sec-samplers}
+
+```{r}
+#| include: false
+source("common.R")
+```
+
+Many quantities of interest in uncertain systems can be expressed as integrals that weight outcomes according to their probability. A common approach to computing these integrals is through Monte Carlo estimation, where we draw samples from our distribution of interest and take their mean as an approximation. This method, known as a 'Monte Carlo' estimate, has been central to probabilistic inference and has driven the development of sophisticated sampling algorithms since the advent of modern computing in the 1950s.
+
+The `monty` package offers a range of sampling algorithms tailored to different types of distributions and varying levels of available information. This flexibility allows users to select the most efficient sampler based on their distribution's characteristics and computational resources.
+
+## Samplers supported by `monty`
```

Comment:
This section reads as a reference to what is in monty - that would belong as a monty vignette and would be great to have.  This section *should* show how different samplers can help solve different problems. It looks like the content here was copied in from the vignettes, so drop for now.